fire/SFMainMod.F90

```diff
@@ -141,7 +141,7 @@ subroutine  fire_danger_index ( currentSite, bc_in)
        d_NI = 0.0_r8
        currentSite%acc_NI = 0.0_r8
     else 
-       yipsolon = (SF_val_fdi_a* temp_in_C)/(SF_val_fdi_b+ temp_in_C)+log(rh/100.0_r8) 
+       yipsolon = (SF_val_fdi_a* temp_in_C)/(SF_val_fdi_b+ temp_in_C)+log(max(1.0_r8,rh)/100.0_r8) 
```

Comment:
I'm surprised that this was necessary.  RH<1 is incredibly dry, almost impossible.  How low are the values seen here?